src/helpers.js

```diff
@@ -1,38 +1,68 @@
 'use babel'
 
-import ChildProcess from 'child_process'
-import { createFromProcess } from 'process-communication'
 import { join } from 'path'
 import escapeHTML from 'escape-html'
 import ruleURI from 'eslint-rule-documentation'
 import { generateRange } from 'atom-linter'
+import cryptoRandomString from 'crypto-random-string'
```

Comment:
We just need something "sufficiently random" to identify each run, a UUID would guarantee it... but that module is rather large in size, and takes 1 second (really!) to initialize, so I used this instead.

We could even just stringify some `Math.random()` output and probably be just fine here if you want to remove the module.